lib/occson/rails/environment_storer.rb

```diff
@@ -0,0 +1,19 @@
+module Occson
+  module Rails
+    class EnvironmentStorer
+      def initialize(content)
+        @content = content
+      end
+
+      def call
+        @content.split("\n").each do |line|
+          next if line.start_with?('#')
+
+          key, value = line.split("=", 2) # @TODO handle wrapped values
```

Comment:
EnvironmentStorer seems to be doing quite a lot more than _storing_ the _environment_, SRP? Maybe `EnvironmentParser` is a different person altogether?